### DISCUSSION TITLE
# Join Us for the Apache PLC4X Community Meeting in Frankfurt/Main! # March 23rd

### DISCUSSION THREAD

**Discussion by ottlukas** [apache/plc4x#1429] [General]
Hello PLC4X Enthusiasts! 🎉

We're excited to announce that the Apache PLC4X project will be holding a **Community Meeting** in **Frankfurt/Main, Germany**, on **March 23rd**. This is a fantastic opportunity for contributors, users, and enthusiasts to come together in person to discuss, collaborate, and have fun!

## Event Details:

- **Date:** March 23rd
- **Location:** Frankfurt/Main, Germany
- **Agenda:**
  - Meet & Greet
  - Project Updates & Roadmap Discussion
  - Hands-on Sessions
  - Fun Activities & More!

Whether you're a seasoned contributor or new to the project, we welcome you to join us. It's a chance to meet the faces behind the usernames, share ideas, and work together on making PLC4X even better.

Don't miss out on the fun and the opportunity to contribute to an open-source project in a meaningful way. Let's get together and get things done!

**RSVP:** Please let us know if you'll be attending by responding to this thread.

Looking forward to meeting you all in person!

Best,
The Apache PLC4X Team

**Comment by splatch** [comment]
I'd like to estimate costs and start working on budget allocation with my chief of staff. Are we aiming Frankfurt Innenstadt II area this time?

**Reply by ottlukas** [reply]
Frankfurt Innenstadt II area  - yes close to Frankfurt Messe station

**Comment by chrisdutz** [comment]
We should mention that the meetup will take place in the Frankfurt codecentric office (Where everything began)

**Reply by ottlukas** [reply]
Indeed this is why we mention that on Linkedin.
